cernopendata/modules/fixtures/data/docs/cms-mit-report-2017/cms-mit-report-2017.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "title": "The Future is Open",
```

Comment:
@atrisovic  What about slugs?  Without a slug this article will be published under the URL `/docs/the-future-is-open`.  This wouldn't follow the usual structure... shall we call it `cms-the-future-is-open-2017` and use it as the slug and in the directory name everywhere?  (rather than `cms-mit-report-2017`, where the word "report" may read a bit weird...)

Reply:
I like the proposed slug.